### PR TITLE
license-control-service: remove use of bindata library

### PR DESCRIPTION
**Pull request by stevendanna**
The use of this library seemed like overkill in this service as we
were getting data from a single file. In this change, I now directly
generate the file with a go template.

Signed-off-by: Steven Danna <steve@chef.io>

### CODE REVIEW

File: components/license-control-service/pkg/keys/keys.go [hunk 1]
```diff
@@ -1,13 +1,11 @@
 package keys
 
 // Generate key bindata
-//go:generate curl --fail --silent --output data/keys.json http://license-acceptance.chef.co/keys
-//go:generate go-bindata -pkg $GOPACKAGE -o keys.bindata.go data/...
+//go:generate go run ../../tools/gen-keys https://license-generation-service.chef.co/keys keys.bindata.go
```
Comment (stevendanna) [inline components/license-control-service/pkg/keys/keys.go]: Note if you try this out locally, you need to be on the VPN.

File: components/license-control-service/tools/gen-keys/main.go [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"text/template"
+	"time"
+
+	"github.com/sirupsen/logrus"
```
Comment (srenatus) [inline components/license-control-service/tools/gen-keys/main.go]: 💭 Does such a small utility need logrus? Fine. 😉

Comment (yzl) [reply components/license-control-service/tools/gen-keys/main.go]: Not directly related to the PR, but...do we *have* a standard for what logging framework to use when?

Comment (srenatus) [reply components/license-control-service/tools/gen-keys/main.go]: I don't think so. But also, I guess I shouldn't scold anyone for using what we use everywhere everywhere 😉 :ignoreme:

**Pull request merged**